examples/workflows/sentiment_example.py

```diff
@@ -76,20 +76,22 @@ class SentimentWorkflow(AbstractWorkflow):
     pass
 
 
-def main(args):
-    config = decode_workflow_token(args.workflow_token)
+def execute(token, logger, worker_number, *args, **kwargs):
```

Comment:
3 arguments, bute execute is called with one

Reply:
typing

Reply:
Good point - resolved. Thanks.